etc/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# Install the bash completion script in an unversioned location.
+install(FILES ign.bash_completion.sh DESTINATION share/ignition/ignition.completion.d)
+install(FILES ignition.completion DESTINATION share/ignition)
```

Comment:
Can all these files and directories be `gz` instead of `ignition`?

Reply:
[1fba3f1](https://github.com/gazebosim/gz-tools/pull/87/commits/1fba3f16d5b71edb5b169b3d00412744a469a9b3)